test/e2e-go/features/transactions/asset_test.go

```diff
@@ -812,37 +829,120 @@ func TestAssetCreateWaitRestartDelete(t *testing.T) {
 		asset = cp
 		assetIndex = idx
 	}
-	a.Equal(asset.UnitName, "test")
-	a.Equal(asset.AssetName, "testname")
-	a.Equal(asset.ManagerAddr, manager)
-	a.Equal(asset.ReserveAddr, reserve)
-	a.Equal(asset.FreezeAddr, freeze)
-	a.Equal(asset.ClawbackAddr, clawback)
-	a.Equal(asset.MetadataHash, assetMetadataHash)
-	a.Equal(asset.URL, assetURL)
+	verifyAssetParameters(asset, "test",  "testunit", manager, reserve, freeze, clawback,
+		assetMetadataHash, assetURL,  a)
 
 	// Destroy the asset
-	txids = make(map[string]string)
-	tx, err = client.MakeUnsignedAssetDestroyTx(assetIndex)
-	sender := manager
-
-	// re-generate wh, since this test takes a while and sometimes
-	// the wallet handle expires.
-	wh, err = client.GetUnencryptedWalletHandle()
+	tx, err := client.MakeUnsignedAssetDestroyTx(assetIndex)
+	submitAndWaitForTransaction(manager, tx, "destroying assets", client, fixture, a)
 
-	txid, err = helperFillSignBroadcast(client, wh, sender, tx, err)
+	// Check again that asset is destroyed
+	info, err = client.AccountInformation(account0)
 	a.NoError(err)
-	txids[txid] = sender
+	a.Equal(len(info.AssetParams), 0)
+	
+	// Should be able to close now
+	wh, err := client.GetUnencryptedWalletHandle()
+	_, err = client.SendPaymentFromWallet(wh, nil, account0, "", 0, 0, nil, reserve, 0, 0)
+	a.NoError(err)
+}
 
-	_, curRound = fixture.GetBalanceAndRound(account0)
-	confirmed = fixture.WaitForAllTxnsToConfirm(curRound+20, txids)
-	a.True(confirmed, "destroying assets")
+
+/** Helper functions **/
+
+// Setup the test and the network 
+func setupTestAndNetwork(t *testing.T, proto string)(
+	Assertions *require.Assertions, Fixture *fixtures.RestClientFixture,  Client *libgoal.Client, Account0 string) {
+
+	t.Parallel()
+	asser := require.New(t)
+	if 0 == len(proto){
+		// If the  protocol is not specified, used the default one
+		proto = "TwoNodes50EachFuture.json"
```

Comment:
don't use "future" anymore. we're already in the future

Reply:
This is already in the test configuration. We need to remove "future" from many places to make it consistent.